README.md

```diff
@@ -92,119 +92,140 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+ - Fixed an issue causing the Calculator plugin to return division by zero errors when dividing by hexadecimal numbers. Thanks [@plante-msft](https://github.com/plante-msft)!
+ - Updated the Calculator plugin Mages library to 3.0.0 and added support for the random integer function. Thanks [@htcfreek](https://github.com/htcfreek)!
+ - Improved handling of non-base 10 numbers to add support for binary and octal numbers in the Calculator plugin. Thanks [@PesBandi](https://github.com/PesBandi)!
+ - Added a setting to enable selection of which units to use for trigonometric functions. Thanks [@OldUser101](https://github.com/OldUser101)!
+ - Fixed a .NET 9 regression causing the PowerToys Run dialog to not be draggable. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - Added context menu buttons for the VS Code Workspaces plugin, for copying the path, opening in File Explorer or in Console. Thanks [@programming-with-ia](https://github.com/programming-with-ia)!
+ - Added some telemetry to gather data on which hotkey is used to trigger PowerToys Run.
+ - Removed the workarounds that were in place to fix some WPF issues that were fixed in .NET 9.0.1.
+ - Fixed a typo in the Value Generator plugin messages. Thanks [@OldUser101](https://github.com/OldUser101)!
 
 ### Quick Accent
 
- - Added a setting to allow selecting which character sets to show. Thanks [@Sirozha1337](https://github.com/Sirozha1337)!
+ - Added the ć character to the Slovenian character set. Thanks [@dsoklic](https://github.com/dsoklic)!
+ - Added the Proto-Indo-European character set. 
 
-### Screen Ruler
+### Registry Preview
 
- - Added a Setting to also allow showing measurements in inches, centimeters or millimeters. Thanks [@Sophanatprime](https://github.com/Sophanatprime)!
+ - Fixed an issue causing line breaks to not be parsed correctly for REG_MULTI_SZ values. Thanks [@htcfreek](https://github.com/htcfreek)!
+ - Added a tooltip to values to show multiple lines of data. Thanks [@htcfreek](https://github.com/htcfreek)!
+ - Added a context menu to enable copying type, value and key paths. Thanks [@htcfreek](https://github.com/htcfreek)!
 
 ### Settings
 
- - Fixed an issue causing all the links to milestones in the "What's new?" OOBE page to point to the same milestone.
- - Removed extra space from the Welcome page. Thanks [@agarwalishita](https://github.com/agarwalishita)!
- - Updated left navigation bar icons. Thanks [@niels9001](https://github.com/niels9001)!
- - Fixed accessibility issues in the dashboard page. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - Made the Advanced Paste paste OpenAI configuration modal scrollable.
+ - Fixed the text on the Quick Accent page to refer to "character sets" instead of "character set". Thanks [@PesBandi](https://github.com/PesBandi)!
+ - Added the plugin version and website to the PowerToys Run plugin settings. Thanks [@htcfreek](https://github.com/htcfreek)!
```

Comment:
Requires correct file version attribute of dlls.

Reply:
I don't get it, is this something you want to change in the description? 😅

Reply:
Just added it, hope that works ;)